src/sagemaker/lineage/query.py

```diff
@@ -300,13 +303,66 @@ def _import_visual_modules(self):
         get_module("pyvis")
         from pyvis.network import Network
         from pyvis.options import Options
+        from IPython.display import IFrame
 
-        return Network, Options
+        get_module("bs4")
+        from bs4 import BeautifulSoup
+
+        return Network, Options, IFrame, BeautifulSoup
```

Comment:
Did these two new imports require additional installations by end customers?

Reply:
I didn't add BeautifulSoup into the [test requirements ](https://github.com/Snoyark/sagemaker-python-sdk/blob/master/requirements/extras/test_requirements.txt) but it still ran successfully on unit and integ tests, does it mean it doesn't require additional installations by end customers?